mono/utils/mono-systemcerts.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+typedef enum
+{
+	DATATYPE_STRING = 0,
+	DATATYPE_INTPTR = 1
+} CertDataFormat;
+
+typedef struct
+{
+	void* certdata;
+	int certsize;
+} CertObj;
+
+int EnumSystemCertificates(void* certStore, void** iter, int *format, int* size, void** data);
```

Comment:
Let's not change these now, but when we move to IL2CPP the C API naming convention is `UnityPal*`